README.md

```diff
@@ -194,7 +195,7 @@ Keycloak is used to authenticate to Pravega on Streaming Data Platform (SDP).
     ```shell
     edge@edge1:~$
     NAMESPACE=edge
-    kubectl get secret ${NAMESPACE}-pravega -n ${NAMESPACE} -o jsonpath="{.data.keycloak\.json}" | base64 -d ; echo
+    kubectl get secret ${NAMESPACE}-ext-pravega -n ${NAMESPACE} -o jsonpath="{.data.keycloak\.json}" | base64 -d ; echo
```

Comment:
Can you confirm that the content of `${NAMESPACE}-pravega` and `${NAMESPACE}-ext-pravega` are the same?

Reply:
In SDP 1.5 and above _${NAMESPACE}-pravega_ will gives us internal endpoint(which can only be accessed inside pod)
ex:- "auth-server-url":"https://keycloak-http.nautilus-system.svc.cluster.local:8443/auth"

where as ${NAMESPACE}-ext-pravega gives us the external ingress which can be used to connect externally too even if its running in local or outside SDP
ex:-"auth-server-url":"https://keycloak.karnak.ns.sdp.hop.lab.emc.com/auth"
(Note:- before SDP 1.4 it used to get external ingress directly when _${NAMESPACE}-pravega_ was used, now it has been changed)